src/index.js

```diff
@@ -8,13 +8,7 @@ const mergeFolders = require('./merge-folders').merge
 function buildBundle (config) {
   return rollup.rollup(config)
     .then(function (bundle) {
-      return bundle.write({
-        format: config.format || 'es6',
```

Comment:
@YuryStrozhevsky I'm always a fan of the less logic the better. Truth be told could even remove the `es6` default unless there is  special reason. However we should notify @bahmutov as this may require a version bump 👊 . Thoughts?

Overall an amazing contribution. And thanks for your time on this!